README.md

```diff
@@ -73,9 +73,10 @@ terraform destroy
 
 ### ERD diagram
 
-[ERD diagram](https://drawsql.app/teams/trainee-4/diagrams/railway-tracker-3nf)
+[ERD diagram](https://drawsql.app/teams/isaac-11/diagrams/railwaytracker-erd)
```

Comment:
I don't think we need to include the drawsql link here, I think just the image is sufficient.

Reply:
The ERD link has been removed.